client/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './App.css';
+import AuthForm from './components/AuthForm';
+import UserList from './components/UserList';
+axios.defaults.withCredentials = true;
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      authed: true,
+      users: [],
+    };
+  }
+  handleRetrieveUsers = () => {
+    axios.get('http://localhost:5000/api/users')
+      .then(res => this.setState({ users: res.data.users }))
+      .catch(err => {
+        this.setState({ authed: false });
+      });
+  }
+  handleLogin = (user) => {
+    axios.post('http://localhost:5000/api/login', user)
+      .then(res => this.setState({ authed: true }, this.handleRetrieveUsers))
+      .catch(err => console.log(err.response.data.error));
+  }
+  handleLogout = ev => {
+    axios.get('http://localhost:5000/api/logout')
+        .then(res => this.setState({ authed: false, users: [] }))
+        .catch(err => console.log(err));
+}
+  handleRegister = (user) => {
+    axios.post('http://localhost:5000/api/register', user)
```

Comment:
It's typical to have your api url (at least, if it's internal, like this one) stored somewhere in your FE; you could use a separate file to export it for use in all files, but in this case a variable declared above your App would do:
```javascript
const API_URL = 'http://localhost:5000/api';
```
and the usage, in your handleRegister method for example:
```javascript
  axios.post(`${API_URL}/register`, user)
  // ...
```